src/app/cube/content-pages/editorial-process/copy.ts

```diff
@@ -117,17 +124,19 @@ export const sections = {
                             <li>Accepted Major</li>
                             <ul>
                                 <li>A learning object is missing files.</li>
-                                <li>A learning object video does not have captioning.</li>
+                                <li>A learning object video is not available on YouTube with closed captions.</li>
                                 <li>A learning object lacks proper contributor attribution.</li>
                                 <li>A learning object file has an incomplete paragraph or section.</li>
-                                <li>A learning object contains some outdated curriculum.</li>
+                                <li>A learning object contains outdated curriculum that impedes the learning outcomes.</li>
+                                <li>A learning object that is not usable based on unavailable content.</li>
                             </ul>
                             <li>Accepted Minor</li>
                             <ul>
-                                <li>A learning object contains assessment solutions.</li>
                                 <li>A learning object has a color contrast issue.</li>
-                                <li>A learning object was submitted to the wrong collection.</li>
+                                <li>A learning object was submitted to the wrong collection</li>
                                 <li>A learning object is under the wrong learning object length.</li>
+                                <li>A learning object is very large (such as a hierarchy) and should be 
```

Comment:
For our users, it is not always clear what a hierarchy is. I think it would be best to make the word hierarchy in `(such as a hierarchy)` a clickable element to the hierarchy section of the about-us page. you would make an a tag like this:
`<a routerLink="/about-us#hierarchies">hierarchy</a>`

![image](https://github.com/Cyber4All/clark-client/assets/55514493/63c3a958-64e7-4be2-b1ce-61343ebeed51)